Tetris Clone C++/TetrisPiece.cpp

```diff
@@ -274,25 +279,30 @@ void TetrisPiece::lock() {
 }
 
 void TetrisPiece::updateRectUsingDelta(int dx, int dy, sf::RectangleShape &shape) {
-    int rect_x, rect_y;
-    const sf::Vector2<float> pos = shape.getPosition();
-    rect_x = (int) pos.x + dx;
-    rect_y = (int) pos.y + dy;
-    shape.setPosition(rect_x, rect_y);
+    shape.move(dx, dy);
```

Comment:
remove wrapper method: `void TetrisPiece::updateRectUsingDelta(int dx, int dy, sf::RectangleShape &shape)` and replace with `shape.move()`